rust/src/index/vector/pq.rs

```diff
@@ -281,6 +246,30 @@ impl ProductQuantizer {
     }
 }
 
+impl From<&pb::Pq> for ProductQuantizer {
+    fn from(proto: &pb::Pq) -> Self {
+        Self {
+            nbits: proto.num_bits,
```

Comment:
nit: don't feel strongly at all but should we just call this `num_bits`

Reply:
k, will do.